conf/messages.en

```diff
@@ -113,6 +113,12 @@ accountingPeriodEndDate.error.required = The End date must include {0}
 accountingPeriodEndDate.error.invalid = Enter a valid End date for the accounting period, like 27 3 2023
 accountingPeriodEndDate.change.hidden = End date
 
+accountingPeriodIrrelevant.title = You're not eligible for Marginal Relief
+accountingPeriodIrrelevant.heading = You're not eligible for Marginal Relief
+accountingPeriodIrrelevant.body = Your accounting period is before the 1 April 2023 start date for Marginal Relief.
```

Comment:
Is this 1st of 2nd April?

Reply:
Not sure this is the exact phrasing I got from the ticket.

Reply:
https://jira.tools.tax.service.gov.uk/browse/MRC-90

Reply:
Please check with Darryl, this must a be a mistake. You test says "before 2nd April"

Reply:
Hmm, just re-read it. The message seems correct, but a bit confusing as it does not refer to start date or end date. Probably leave it as it is

Reply:
Messaged Darryl to double check this. Yeah the page texts talks about the period. I think it's right. I agree, the wording  is a bit awkwardly phrased. I'll see if Darryl has an opition about this and proceed.